tasks/task757_msr_sqa_question_generation.json

```diff
@@ -0,0 +1,6347 @@
+{
+    "Contributors": [ "Shreya Sarnaik sjsarnai@asu.edu" ],
+    "Categories": [ "Question Generation" ],
+    "Source": [ "Microsoft Research Sequential Question Answering (Dataset - https://huggingface.co/datasets/msr_sqa#source-data) (Paper - https://www.microsoft.com/en-us/research/wp-content/uploads/2017/05/acl17-dynsp.pdf)" ],
+    "Definition": "In this task, you are given a passage (data in tabular/json form) and you must generate a question based on the information present in the passage. Note that the passage is given in tabular format. Construct a question in such a way that 1) It is unambiguous, 2) It is answerable from the passage, 3) Its answer is unique, 4) Its answer is a continuous text span from the passage. Avoid creating questions that 1) Can be answered correctly without reading/understanding the passage, 2) Uses same phrases/words form the given passage.",
```

Comment:
Why do we need to mention "passage"? Just cut to the chase: 
```
information present in the passage. Note that the passage is given in tabular format
```
--> 
```
information present in tabular format...
```

Reply:
And similarly for the rest of the tasks.

Reply:
@danyaljj @swarooprm , Should I make the changes?